src/traces/parcoords/attributes.js

```diff
@@ -141,6 +141,23 @@ module.exports = {
         description: 'The dimensions (variables) of the parallel coordinates chart. 2..60 dimensions are supported.'
     }),
 
+    unselected: {
+        line: {
+            color: {
+                valType: 'color',
+                dflt: '#777',
+                role: 'style',
+                editType: 'plot',
+                description: [
+                    'Sets the color of lines in the background',
+                    'i.e. unselected lines.'
+                ].join(' ')
+            },
+            editType: 'calc'
+        },
+        editType: 'calc'
```

Comment:
if `color` has `editType: 'plot'` and it's the only thing in these containers, they can have `'plot'` as well.

Reply:
I recall, Etienne mentioned that for containers we should use `calc`.

Reply:
I believe containers should use the most intense `editType` of anything inside them, and should use `calc` if there's any array or `arrayOk` attribute inside.

Reply:
Thanks for the note regarding `arrayOk` items.
Changed all to `plot` in https://github.com/plotly/plotly.js/pull/4635/commits/b4dabb224c37724951f638462dcb1421ab85011e.